colors.scss

```diff
@@ -478,17 +478,17 @@ $ui-background: $blk-010;
 $ui-foreground: $blk-190;
 
 // UI Colors Dark Theme
-$ui-blue-dark: $v-bb-140;
-$ui-blue-hover-dark: $v-bb-120;
-$ui-blue-pressed-dark: $v-bb-160;
-$ui-green-dark: $v-gg-140;
-$ui-green-hover-dark: $v-gg-120;
-$ui-green-pressed-dark: $v-gg-160;
-$ui-yellow-dark: $v-yy-140;
-$ui-yellow-hover-dark: $v-yy-120;
-$ui-yellow-pressed-dark: $v-yy-160;
-$ui-red-dark: $v-rr-140;
-$ui-red-hover-dark: $v-rr-120;
-$ui-red-pressed-dark: $v-rr-160;
-$ui-background-dark: $blk-190;
-$ui-foreground-dark: $blk-010;
\ No newline at end of file
+$ui-blue-dark: #00A0FF;
+$ui-blue-hover-dark: #0087D7;
+$ui-blue-pressed-dark: #47BBFF;
+$ui-green-dark: #36DA43;
+$ui-green-hover-dark: #11AD1D;
+$ui-green-pressed-dark: #44ED51;
+$ui-yellow-dark: #FFC900;
+$ui-yellow-hover-dark: #F4B000;
+$ui-yellow-pressed-dark: #44ED51;
+$ui-red-dark: #FE583E;
+$ui-red-hover-dark: #F3381B;
+$ui-red-pressed-dark: #FF6852;
+$ui-background-dark: $blk-210;
+$ui-foreground-dark: $blk-000;
```

Comment:
I think the foreground is the wrong variable here -  "dark theme" ui-foreground: `$blk-200` (#2b2b2b) and ui-background: `$blk-210` (#202020)

Reply:
The way we are using foreground is for text. Look at light theme foreground and background. That’s how Paul and I initially put it together.

Reply:
Yeah, that's just kind of confusing no? I'd expect background to be the ... background and foreground to... be the foreground? Wouldn't that also limit us to one "named" text color?

Reply:
I'm good to change that variable if it's confusing. Just wanted to create a var for normal body text color. Open to suggestions

Reply:
oh, if its normal body text color than we should call it that. its misleading fo sho. if we are following #555555 for light theme body, then we should go #bfbfbf for dark theme body

Reply:
e.g.

Light theme:
#2b2b2b headers
#555555 body

Dark theme:
#ffffff headers
#BFBFBF body

Reply:
ui-foreground: `$blk-000`
ui-foreground-hover: `$blk-010`
ui-foreground-pressed: `$blk-020`

ui-text-1: `$blk-160`
ui-text-2:`$blk-180`
ui-text-3: `$blk-200`

ui-foreground-dark: `$blk-200`
ui-foreground-dark-hover: `$blk-210`
ui-foreground-dark-pressed: `$blk-220`

ui-text-dark-1: `$blk-060`
ui-text-dark-2:`$blk-040`
ui-text-dark-3: `$blk-020`

Kind of confusing to have "dark" be light and vice versa, but those are the kind of variables I'd want to use, a range of "surface" colors for default, hover, pressed, and then a few body text colors (or just one if we think three is two many)...

Reply:
Yeah, "header / body/ ?" better than levels of 1/2/3 fo sho.

Reply:
@macandcheese for this comment: 

ui-foreground-hover: $blk-010
ui-foreground-pressed: $blk-020

ui-foreground-dark-hover: $blk-210
ui-foreground-dark-pressed: $blk-220

down and down. love this